<?xml version="1.0"?>
<launch>
    <group ns="$(env DRONE_NAME)">
        <node pkg="borealis_helper_test_funcs" type="uwb_t265_republisher.py" name="uwb_t265_republisher" output="screen"/>
        <remap from="/uwb_uav_setpoint" to="uwb_setpoint"/>
        <remap from="/t265_uav_setpoint" to="mavros/setpoint_position/local"/>
    </group>
</launch>
